domino-ui/src/main/java/org/dominokit/domino/ui/forms/BasicFormElement.java

```diff
@@ -351,7 +351,9 @@ public T setRequiredErrorMessage(String requiredErrorMessage) {
   /** {@inheritDoc} */
   @Override
   public String getRequiredErrorMessage() {
-    return isNull(requiredErrorMessage) ? "* This field is required." : requiredErrorMessage;
```

Comment:
The default message should go as a default in the DominoUIConfig

Reply:
I'll change the PR. Please check.